pkg/cluster/namespace.go

```diff
@@ -19,7 +19,7 @@ type NamespaceRequest struct {
 
 	// array of functions to mutate role before returning to requester
 	Mutations []mutation.MutateFunc
-	Client    interface{}
+	Client    ctrlClient.Client
```

Comment:
Can we name controller a better way? Probably `cntlr`? `ctrl` makes me feel it's `control`. I know it's from the existing name but it doesn't follow a clear abbreviation rule.

ref: https://www.abbreviations.com/abbreviation/controller

Reply:
updated